render-test/manifest_parser.cpp

```diff
@@ -414,7 +434,14 @@ mbgl::optional<Manifest> ManifestParser::parseManifest(const std::string& manife
             }
 
             if (testPath.path().filename() == "style.json") {
-                testPaths.emplace_back(testPath, getTestExpectations(testPath, path, expectationPaths));
+                const auto defaultExpectationPath{std::move(mbgl::filesystem::path(testPath).remove_filename())};
+                static const auto rootLength = manifest.testRootPath.length();
```

Comment:
`static` is not needed here, otherwise we won't be able to run several manifests within the same process.